File: Admin/Admin/Models/AssClasse.cs
namespace Admin.Models
{
    public class AssClasse
    {
        public int Id { get; set; }
        public int ClasseId { get; set; }


        public Classe? Classe { get; set; }

        public int AssignmentId { get; set; }
        public Assignment? Assignment { get; set; }

        public string ToString(object? obj)
        {
            return "Id: " + Id + " ClasseId: " + ClasseId + " AssignmentId: " + AssignmentId;
        }
    }
}

File: mobile/app/src/main/java/com/example/eduapp/etudiant/models/AssClasse.java
package com.example.eduapp.etudiant.models;

public class AssClasse {
    public int id;
    public int classeId;

    public Classe classe;

    public int assignmentId;
    public Assignment assignment;

    public String toString(Object obj) {
        return "Id: " + id + " ClasseId: " + classeId + " AssignmentId: " + assignmentId;
    }
}

